broker/lib/fabrik/CfPlatformManager.js

```diff
@@ -69,7 +69,7 @@ class CfPlatformManager extends BasePlatformManager {
       .tap(() => logger.info('+-> Security group exists'))
       .catch(SecurityGroupNotFound, () => {
         logger.warn('+-> Security group does not exist. Trying to create it again.');
-        return this.ensureTenantId(options.context.space_guid)
+        return this.ensureTenantId(options)
```

Comment:
One more fix we have to do ensureTenantId(options) {
    return Promise
      `.try(() => _.get(options, 'context.space_guid') ? options.context.space_guid : this.cloudController`
        .getServiceInstance(options.guid)
        .then(instance => instance.entity.space_guid)
      );
  }